int32 indexCategory 
int32 indexChannel 
float32 value 
int32 layer